test/real_world_cases/ox_protocol_test.exs

```diff
@@ -40,18 +40,18 @@ defmodule ExWeb3EcRecover.RealWorldCases.OxProtocolTest do
       domain: %{
         "name" => "0x Protocol",
         "version" => "3.0.0",
-        "chainId" => 137,
+        "chainId" => "137",
         "verifyingContract" => "0xfede379e48c873c75f3cc0c81f7c784ad730a8f7"
       },
       message: %{
         "makerAddress" => "0x1bbeb0a1a075d870bed8c21dfbe49a37015e4124",
         "takerAddress" => "0x0000000000000000000000000000000000000000",
         "senderAddress" => "0x0000000000000000000000000000000000000000",
         "feeRecipientAddress" => "0x0000000000000000000000000000000000000000",
-        "expirationTimeSeconds" => 1_641_635_545,
-        "salt" => 1,
-        "makerAssetAmount" => 1,
-        "takerAssetAmount" => 50_000_000_000_000_000,
+        "expirationTimeSeconds" => "1641635545",
+        "salt" => "1",
+        "makerAssetAmount" => "1",
+        "takerAssetAmount" => "50000000000000000",
```

Comment:
Do we want a separate test case with strings as opposed to changing it?

Reply:
Welp. I changed it purely because this reflects our "Real World Case", I should add additional cases tho...

Reply:
Added additional test.